app/assets/stylesheets/mobile/modal.scss

```diff
@@ -135,6 +135,35 @@
   }
 }
 
```

Comment:
there seems to be a fair bit in common with mobile and desktop here, can any of the duplicate stuff be moved to modal common or something?